MyCryptoHQ/MyCrypto
# Update Create Wallet Tutorial Screenshots

Closes #1537

### Description

Conditionally shows different screenshots if you're on the app vs web

### Steps to Test

1. Generate a wallet on web downloadable, confirm instructions are web-based
2. Generate a wallet on electron, confirm instructions are electron based

### Screenshots

<img width="1223" alt="screen shot 2018-04-23 at 5 10 06 pm" src="https://user-images.githubusercontent.com/649992/39153800-3e40b1e8-471a-11e8-9941-d99821b1dba0.png">
